app/javascript/vue/components/Filter/Facets/shared/FacetYears.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <FacetContainer>
+    <h3> {{ title }} </h3>
+    <div class="field">
+      <label>Year:</label>
+      <br />
+      <input
+        type="number"
+        :min="minYear"
+        :max="maxYear"
+        v-model="params[paramYear]"
+      />
+    </div>
+
+    <div class="margin-medium-bottom">
+      Or
+    </div>
+
+    <fieldset>
+      <legend>Range</legend>
+      <div class="field">
+        <label>After:</label>
+        <br />
+        <input
```

Comment:
import { vBetweenNumbers } from '@/directives/betweenNumbers'

We got a directive for this, you can use `v-between-numbers="[minYear, maxYear]"`

It will force the user to set a number in the range